doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1077,6 +1076,16 @@ DFU libraries
 
 * Added a new DFU SMP target for the image update to an external MCU by using the MCUmgr SMP Client.
 
+Peripheral CPU DFU (PCD) library
+--------------------------------
+
+* Added a new function for reading peripherals CPU version number.
+
+Netboot
```

Comment:
Which library do you want to refer to?

Reply:
I want to refer to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/nrf5340/netboot/README.html

Reply:
Well, in that case, the entry is in wrong place. It should be under Samples -> nRF4340 samples, something like:

* :ref:`nc_bootloader` sample:

  * Write your change here